src/Groups/matrices/form_group.jl

```diff
@@ -729,6 +729,30 @@ function isometry_group(L::ZZLat; algorithm = :direct, depth::Int = -1, bacher_d
   end::MatrixGroup{QQFieldElem, QQMatrix}
 end
 
+automorphism_group(L::ZZLat, v::Vector{QQFieldElem}) = isometry_group(L, matrix(QQ,1,degree(L),v))
+
+"""
+    automorphism_group(L::ZZLat, v::QQMatrix; kwargs...) -> MatrixGroup
+
+Return the stabilizer of the matrix ``v`` in the orthogonal group of ``L``.
```

Comment:
Just wondering: If this function returns a stabilizer, then why is it called `automorphism_group` and not e.g. `stabilizer` ?

Reply:
Because it is a finite stabilizer in an infinite group I do not have direct access to, I cannot simply call `stabilizer(group, vector)`.
Another reason is that it is called the same in magma:
https://magma.maths.usyd.edu.au/magma/handbook/text/349#3451